pkgs/by-name/ra/rainfrog/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  rustPlatform,
+}:
+let
+  version = "0.2.3";
+in
+rustPlatform.buildRustPackage {
+  inherit version;
+  pname = "rainfrog";
+
+  src = fetchFromGitHub {
+    owner = "achristmascarl";
+    repo = "rainfrog";
+    rev = "v${version}";
+    hash = "sha256-2eadWYQbCRd9HkTvzwMo6qeCoFB6359TUpLFY5wkl7s=";
+  };
+
+  cargoHash = "sha256-PVSnooNPzaM96iX7BkYI2r9ZtK19plonzbC8RW/jcNU=";
+
+  passthru.updateScript = nix-update-script { };
```

```suggestion
  passthru = {
    updateScript = nix-update-script { };
    version = testers.testVersion { package = rainfrog; };
  }:
```
You may also try this.

---

Weird, and what happens if you manually run `rainforg --version` ? Is this actually supported at all ? Maybe it's `-V` or something else.

---

The test fails because `VERGEN` info has not been filled properly:
```
❮ ./result/bin/rainfrog --version
rainfrog 0.2.4-VERGEN_IDEMPOTENT_OUTPUT (1980-01-01)

Authors: achristmascarl <carl@rainfrog.dev>

Config directory: /home/gaetan/.config/rainfrog
Data directory: /home/gaetan/.local/share/rainfrog
```

For sure, we can merge without this.

---

How about specifying `RAINFROG_DATA`? https://github.com/NixOS/nixpkgs/pull/357443